packages/marko-web-algolia-site-search/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@ac-business-media/marko-web-algolia-site-search",
+  "description": "Algoli site search components for BaseCMS websites.",
+  "version": "0.0.0",
```

Comment:
This version number should be set to the current value found in the root `lerna.json` file